docs/reference/SourceDocsLib/structs/DocumentOptions.md

```diff
@@ -0,0 +1,92 @@
+**STRUCT**
+
+# `DocumentOptions`
+
+```swift
+public struct DocumentOptions
+```
+
+## Properties
+### `spmModule`
+
+```swift
+public let spmModule: String?
+```
+
+### `moduleName`
+
+```swift
+public let moduleName: String?
+```
+
+### `linkBeginningText`
+
+```swift
+public let linkBeginningText: String
+```
+
+### `linkEndingText`
+
+```swift
+public let linkEndingText: String
+```
+
+### `inputFolder`
+
+```swift
+public let inputFolder: String
+```
+
+### `outputFolder`
+
+```swift
+public let outputFolder: String
+```
+
+### `minimumAccessLevel`
+
+```swift
+public let minimumAccessLevel: AccessLevel
+```
+
+### `includeModuleNameInPath`
+
+```swift
+public let includeModuleNameInPath: Bool
+```
+
+### `clean`
+
+```swift
+public let clean: Bool
+```
+
+### `collapsibleBlocks`
+
+```swift
+public let collapsibleBlocks: Bool
+```
+
+### `tableOfContents`
+
+```swift
+public let tableOfContents: Bool
+```
+
+### `xcodeArguments`
+
+```swift
+public let xcodeArguments: [String]
+```
+
+## Methods
+### `init(spmModule:moduleName:linkBeginningText:linkEndingText:inputFolder:outputFolder:minimumAccessLevel:includeModuleNameInPath:clean:collapsibleBlocks:tableOfContents:xcodeArguments:)`
+
+```swift
+public init(spmModule: String?, moduleName: String?,
+            linkBeginningText: String, linkEndingText: String,
+            inputFolder: String, outputFolder: String,
+            minimumAccessLevel: AccessLevel, includeModuleNameInPath: Bool,
+            clean: Bool, collapsibleBlocks: Bool, tableOfContents: Bool,
+            xcodeArguments: [String])
+```
```

Comment:
Probably worth adding actual docs to these 😆

Reply:
Funny. I actually added your comments from your PR, but only _after_ generating the docs 🤦‍♂️. Will update.